src/components/EpicButtons.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { css } from 'emotion';
+import { space } from '@guardian/src-foundations';
+import { Button } from './Button';
+import { EpicTracking } from './ContributionsEpicTypes';
+import { Cta, Variant } from '../lib/variants';
+import { addTrackingParams } from '../lib/tracking';
+import { addRegionIdToSupportUrl } from '../lib/geolocation';
+
+const buttonWrapperStyles = css`
+    margin: ${space[6]}px ${space[2]}px ${space[1]}px 0;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+
+    &.hidden {
+        display: none;
+    }
+`;
+
+const paymentImageStyles = css`
+    display: inline-block;
+    width: auto;
+    height: 25px;
+    margin: ${space[1]}px 0;
+`;
+
+const buttonMargins = css`
+    margin: ${space[1]}px ${space[2]}px ${space[1]}px 0;
+`;
+
+const PrimaryCtaButton = ({
+    cta,
+    tracking,
+    countryCode,
+}: {
+    cta?: Cta;
+    tracking: EpicTracking;
+    countryCode?: string;
+}): JSX.Element | null => {
+    if (cta) {
+        const buttonText = cta.text || 'Support The Guardian';
+        const baseUrl = cta.baseUrl || 'https://support.theguardian.com/contribute';
+        const urlWithRegion = addRegionIdToSupportUrl(baseUrl, countryCode);
+        const urlWithRegionAndTracking = addTrackingParams(urlWithRegion, tracking);
+
+        return (
+            <div className={buttonMargins}>
+                <Button onClickAction={urlWithRegionAndTracking} showArrow>
+                    {buttonText}
+                </Button>
+            </div>
+        );
+    } else {
+        return null;
```

Comment:
Minor, but I would have this `return` outside of the if/else statement as a default.